src/BucketeerProvider.tsx

```diff
@@ -0,0 +1,83 @@
+import React, { useState, useEffect, type JSX, useMemo } from 'react';
+import { initializeBKTClient } from '@bucketeer/js-client-sdk';
+import type { BKTClient, BKTUser, BKTConfig } from '@bucketeer/js-client-sdk';
+import { BucketeerContext } from './context';
+import { SOURCE_ID_REACT, SOURCE_ID_REACT_NATIVE } from './SourceId';
+
+export async function initializeBKTClientForReact(
+  config: BKTConfig,
+  user: BKTUser
+): Promise<void> {
+  if (
+    config.wrapperSdkSourceId === SOURCE_ID_REACT_NATIVE ||
+    config.wrapperSdkSourceId === SOURCE_ID_REACT
```

Comment:
You check both source IDs here because the React Native SDK also wraps the React SDK, right?